agent/app/agent_capability.go

```diff
@@ -197,6 +201,28 @@ func (agent *ecsAgent) appendTaskENICapabilities(capabilities []*ecs.Attribute)
 	return capabilities
 }
 
+func (agent *ecsAgent) appendVolumeDriverCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
+	// for non-standardized plugins, call docker pkg's plugins.Scan()
+	nonStandardizedPlugins, err1 := agent.scanPlugins()
+	if err1 == nil {
```

Comment:
nit: I don't think we need different error names since they are in the same scope and will be overwritten.